sdk/python/foundation-models/system/content-safety/deploy_with_content_safety.ipynb

```diff
@@ -0,0 +1,803 @@
+    "    credential.get_token(\"https://management.azure.com/.default\")\n",
+    "except Exception as ex:\n",
+    "    # Fall back to InteractiveBrowserCredential in case DefaultAzureCredential not work\n",
+    "    credential = InteractiveBrowserCredential()\n",
+    "\n",
+    "client = CognitiveServicesManagementClient(credential, subscription_id)\n",
+    "# create a new Cognitive Services Account\n",
+    "# name of the AACS to create\n",
+    "name_of_aacs = f\"{endpoint_name}-aacs\"\n",
+    "kind = \"ContentSafety\"\n",
+    "sku_name = \"S0\"\n",
+    "parameters = Account(sku=Sku(name=sku_name), kind=kind, location=aacs_location, properties= AccountProperties(custom_sub_domain_name=name_of_aacs, public_network_access=\"Enabled\"))\n",
+    "# How many seconds to wait between checking the status of an async operation.\n",
+    "wait_time = 10\n",
+    "\n",
+    "poller = client.accounts.begin_create(resource_group_name, name_of_aacs, parameters)\n",
+    "while (False == poller.done()) :\n",
+    "    print (\"Waiting {wait_time} seconds for operation to finish.\".format (wait_time = wait_time))\n",
+    "    time.sleep (wait_time)\n",
+    "    # This will raise an exception if the server responded with an error.\n",
+    "    result = poller.result()\n",
+    "\n",
+    "\n",
+    "print(\"Resource created.\")\n",
+    "\n",
+    "aacs=client.accounts.get(resource_group_name, name_of_aacs)\n",
+    "aacs_endpoint = aacs.properties.endpoint\n",
+    "aacs_resource_id = aacs.id\n",
+    "print(aacs_endpoint)\n",
+    "print(aacs_resource_id)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5. Create LLaMa Oneline endpoint"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 5.1 Decide on SKU and instance count for the LLama Oneline endpoint."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "compute_sku_for_llama=\"Standard_DS5_v2\" # the sku of the compute instance for LLaMa endpoint\n",
+    "compute_instance_count_for_llama=1 # the number of compute instance\n",
+    "llama_endpoint_name=f\"{endpoint_name}-llama\"\n",
+    "print(f\"Will create LLaMa endpoint {llama_endpoint_name} using {compute_instance_count_for_llama} {compute_sku_for_llama} compute instance(s)\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 5.2 Check if LLaMa model is available in the aml registry."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_name = \"gpt2\" # TODO(mingtwan) change to LLaMa\n",
+    "auth_mode_for_llama = \"aml_token\" # we will use AML token for authentication\n",
+    "registry_ml_client = MLClient(credential, registry_name=\"azureml\")\n",
+    "version_list = list(registry_ml_client.models.list(model_name))\n",
```

Comment:
We should do a get by name instead of a list search

Reply:
I tried that approach, and it seems that the `get` operation requires a `version` or `label`. Therefore, I have to use the list operation by model name to get the latest version. In fact, the list operation by model name is just to list the available versions for the given model.